src/terminal/input/terminalInput.hpp

```diff
@@ -43,10 +43,23 @@ namespace Microsoft::Console::VirtualTerminal
 
 #pragma region MouseInput
         // These methods are defined in mouseInput.cpp
+
+        // This magic flag is "documented" at https://msdn.microsoft.com/en-us/library/windows/desktop/ms646301(v=vs.85).aspx
+        // "If the high-order bit is 1, the key is down; otherwise, it is up."
+        static constexpr short KeyPressed{ gsl::narrow_cast<short>(0x8000) };

```

Comment:
I'd maybe move this flag to windowio.cpp, since it seems specific to the win32 implementation